controlpanel/api/rules.py

```diff
@@ -29,6 +29,18 @@ def is_app_admin(user, obj):
     if obj is None:
         return True
 
+    if is_superuser(user):
+        return True
```

Comment:
I see why this was added.

It makes me wonder if this is the right thing to do semantically, e.g. I may be a super user but not an admin of a certain app. 

A part from the semantic of it, maybe there are not side effects of returning true for super admins here.

The annoying thing of not adding this here is that the predicated which was changed from

```python
add_perm('api.add_app_admin', is_authenticated & is_superuser)
```

to 

```python
add_perm('api.add_app_admin', is_authenticated & is_app_admin)
```

would have to have an OR to maintain compatibility for superusers if you know what I mean.

This is probably the simplest solution but I thought it was worth highlight this.